_stbt/core.py

```diff
@@ -1311,26 +1311,29 @@ def is_screen_black(self, frame=None, mask=None, threshold=None,
 
         _region = Region.intersect(_image_region(frame), region)
         greyframe = cv2.cvtColor(crop(frame, _region), cv2.COLOR_BGR2GRAY)
-        _, greyframe = cv2.threshold(
-            greyframe, threshold, 255, cv2.THRESH_BINARY)
-        _, maxVal, _, _ = cv2.minMaxLoc(greyframe, mask.image)
+        if mask.image is not None:
+            cv2.bitwise_and(greyframe, mask.image, dst=greyframe)
+        maxVal = greyframe.max()
```

Comment:
I wonder if this has an effect on performance?  In theory it has to do less work.  In practice it all depends on how optimised numpy is vs opencv for this operation.